### PR TITLE
Add C++ SDK build & test github action

### PR DESCRIPTION
**Pull request by argerus**
Signed-off-by: John Argérus <fixed-term.john.argerus@se.bosch.com>

### CODE REVIEW

**Comment by argerus** [comment]
@SebastianSchildt Do you know why I can't add reviewers? Or am I not supposed to do that?

**Comment by argerus** [comment]
FYI
The added github action wasn't triggered here but in the originating repo. Adding a [link](https://github.com/boschglobal/iot-event-analytics/runs/2303802082#step:6:79) for easier access...

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 [![Gitter](https://badges.gitter.im/iot-event-analytics/community.svg)](https://gitter.im/iot-event-analytics/community)
 [![License](https://img.shields.io/badge/License-MPL%202.0-blue.svg)](https://opensource.org/licenses/MPL-2.0)
+[![C++ SDK](https://github.com/boschglobal/iot-event-analytics/actions/workflows/build_cpp_sdk_cmake.yml/badge.svg?branch=feature%2Fgithub_actions_build_cpp_sdk)](https://github.com/boschglobal/iot-event-analytics/actions/workflows/build_cpp_sdk_cmake.yml)
```
Comment (SebastianSchildt) [inline README.md]: I think this should point to the upstream repo. (genivi)

**Review by SebastianSchildt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (argerus) [reply README.md]: I think so too :)

**Review by SebastianSchildt — APPROVED** [review]
Looking fine. lgtm

**Pull request merged**